packages/commonwealth/client/scripts/views/menus/create_content_menu.tsx

```diff
@@ -235,7 +236,7 @@ const getCreateContentMenuItems = (navigate): PopoverMenuItem[] => {
           {
             label: 'New Thread',
             onClick: () => {
-              resetSidebarState();
+              isMobile && resetSidebarState();
```

Comment:
This fixes it for the "New Thread" option, but I am still able to reproduce it with other options like "New proposal" from the header and other options from the app sidebar. This is on a 13.3-inch (2560 × 1600) display (with the dev panel open to half window height).

I also did a quick search for `resetSidebarState()` references and there seem to be a good number of these (not sure if the issue happens in all of those places)

https://github.com/hicommonwealth/commonwealth/assets/51641047/963e12d2-08a7-4e6a-9eff-deebabaa0f80

<img width="305" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/51641047/7d148f03-b5f3-4d30-b494-fb5762129fb0">

Reply:
for context, some of these are intentional, for example when clicking explore menu community to community

Reply:
Just added a fix that follows the DRY principle, i clicked around all the buttons i could find and it seems this is fixed now.